transformer/mappings.py

```diff
@@ -112,20 +84,35 @@ def title(self, value):
     def type(self, value):
         return value.split('note_',1)[1]
 
-    @odin.map_list_field(from_field='subnotes', to_field='subnotes', to_list=True)
-    def subnotes(self, value):
-        if value:
-            return (ArchivesSpaceSubnoteToSubnote.apply(v) for v in value)
-
-    @odin.map_field(from_field='content', to_field='subnotes', to_list=True)
-    def subnotes(self, value):
-        if value:
-            return Subnote(type='text', content=value)
```

Comment:
Here I have removed two mappings that were pointing to `subnotes`, which seemed to be the cause of the initial failure of the mapping to produce anything.